posthog/temporal/workflows/metrics.py

```diff
@@ -0,0 +1,25 @@
+import temporalio
+
+
+def get_rows_exported_metric(destination: str) -> temporalio.common.MetricCounter:
+    return temporalio.activity.metric_meter.with_additional_attributes({"destination": destination}).create_counter(
+        "batch_export_rows_exported", "Number of rows exported."
+    )
```

Comment:
I verified that `create_counter` (and others) are reusable, i.e. they point into the metrics registry and can be reused. This is necessary because we're no longer registering metrics in global scope. We use the `activity` to reach the runtime and create a metric as needed to increment.

Anyway, you can keep "creating" the same metric and incrementing it, and it's actually incrementing a single number deep down in the registry server.